backend/api/src/test/kotlin/software/shonk/application/service/ShorkServiceTest.kt

```diff
@@ -346,4 +346,39 @@ class ShorkServiceTest {
         assertEquals(Winner.DRAW, result.result.winner)
         assertTrue(result.visualizationData.isEmpty())
     }
+
+    @Test
+    fun `test get lobby settings for a valid lobby`() {
+        val shorkService = ShorkService(Shork())
+        val lobbyId = shorkService.createLobby("playerA").getOrThrow()
+        shorkService.setLobbySettings(lobbyId, Settings())
+        val result = shorkService.getLobbySettings(lobbyId)
+        val defaultSettings = Settings()
+        val testSettings = result.getOrThrow()
+
+        assertTrue(result.isSuccess)
```

Comment:
isn't this assert redundant as above you already call getOrThrow() which would throw an Exception is the result is failure?

Reply:
I tend to be overly cautious with the assertions in testing that I didn't notice that this assertion is actually redundant